### PR TITLE
Added replicate the node process for performance section

### PR DESCRIPTION
**Pull request by migramcastillo**
I didn't see anything on the discussion issue to solve, so here is the contribution for the performance section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
 
 <br/><br/><br/>
 
+## ![✔] 7.2. Replicate the Node process
+
+ **TL;DR:** If you really need to use some blocking/heavy compute code in your application, you can’t use any of the recommendation about [non-blocking the event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/) and you have a managed server with more than a single CPU, you should consider using the Cluster Module native from NodeJS or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode/). Cluster mode will allow you to take advantage of your CPU resources to replicate the Single Thread having availability when the Event Loop is blocked.
```
Comment (BrunoScheufler) [inline README.md]:
```suggestion
 **TL;DR:** If you really need to use some heavy computational work in your application and you can’t make use of the [non-blocking event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop), and if you have a managed server with more than a single CPU, you should consider using the native Node.js `cluster` module or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode). Cluster mode will allow you to take advantage of your CPU resources to replicate the single thread when the Event Loop is blocked.
```

**Comment by migramcastillo** [comment]
> Looks like a great change 💯 I’ve added a suggestion regarding our style guidelines which might be more inline with our existing content 👍

Thanks for the suggestion!

File: README.md [hunk 2]
```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
 
 <br/><br/><br/>
 
+## ![✔] 7.2. Replicate the Node process
+
+ **TL;DR:** If you really need to use some heavy computational work in your application and you can’t make use of the [non-blocking event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop), and if you have a managed server with more than a single CPU, you should consider using the native Node.js `cluster` module or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode). Cluster mode will allow you to take advantage of your CPU resources to replicate the single thread when the Event Loop is blocked.
```
Comment (i0natan) [inline README.md]: Replication gains better performance even for IO-only work - I tried once to run 1 vs 4 express apps that just serve requests (no CPU work) and saw 20% improvements for 4 processes

I would generalize the advice - if you have more than 1 vCPU, replicate a Node.js process per each

Comment (i0natan) [inline README.md]: We want to modernize this guide and refer to modern platform (k8s, serverless). I would start there and mention that most modern platforms have build-in replication capabilities, in case you deploy to a bare server then consider using some... (cluster, PM2)

File: README.md [hunk 3]
```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
...
+
+**Otherwise:** In high peak traffic your users could suffer some latency when the Event Loop is blocked and if you have a multi CPU managed server you will be wasting valuable resources.
```
Comment (i0natan) [inline README.md]: Sums it up well, consider spicing it up - You pay for 4 cpu and starve 3 of them.

File: README.md [hunk 4]
```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
...
+
```
Comment (i0natan) [inline README.md]: There should be 'Read more' section
